src/tickets/TicketController.ts

```diff
@@ -0,0 +1,118 @@
+import { myFormattedDate } from "@app/helper/FormattedDate";
+import "reflect-metadata";
+import {
+  Body,
+  Controller,
+  Get,
+  HttpError,
+  Param,
+  Post,
+  Put,
+} from "routing-controllers";
+import { Container } from "typedi";
+import { TicketStatus } from "../entities/ITicket";
+import { TicketService } from "./TicketService";
+import {
+  PostStatusRequest,
+  UpdateTicketRequest,
+  UpdateTicketStatusRequest,
+} from "./dto/TicketRequest";
+
+@Controller()
+export class TicketController {
+  @Get("/tickets")
+  async getAll() {
+    try {
+      const serviceinstant = Container.get(TicketService);
+      const alltick = await serviceinstant.getAllTickets();
+      return alltick;
+    } catch (err) {
+      if (err instanceof HttpError) {
+        console.error(err);
+        throw new HttpError(err.httpCode, err.message);
+      }
```

Comment:
if throw error doesn't really for specific problem don't have to do it use only throw error or throw generic error 
```suggestion
      if (err instanceof HttpError) {
        throw new HttpError(err.httpCode, err.message);
      } else {
        throw new HttpError(500, "Something went wrong, please contact admin");
      }
```